### ISSUE TITLE
When exporting as html, math is not rendered at all on other computers

### ISSUE THREAD

**Issue by StHante** [shd101wyy/markdown-preview-enhanced#259]
I found that when I export a document containing maths, the maths is not rendered, when I copy the produced html file to different computers and try to open them there. I investigated and found the following:
The html file links to the local copy of MathJax:

`<script type="text/javascript" async src="file:///home/myuser/.atom/packages/markdown-preview-enhanced/dependencies/mathjax/MathJax.js?config=TeX-AMS_CHTML"></script>`

The rendering will work fine, when I replace the above link by the online version:

`<script src='https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_CHTML'></script>`

Maybe this could be fixed, so I don't have to replace it every time.

Great package!

**Comment by shd101wyy** [comment]
Hi @StHante,

Make sure you have 'cdn' option checked when exporting HTML.

Thanks

**Comment by StHante** [comment]
Thanks for the quick reply. Maybe that button should be relabeled, because I honestly have no idea what "CDN" stands for and I'm probably not the only one. Tremendous plugin!

**Comment by shd101wyy** [comment]
Any suggestion for renaming the label?

**Comment by StHante** [comment]
Maybe something like "Use online copy of MathJax". (Assuming it only affects the MathJax, which I am not sure of)
